Intersect.Client.Core/Localization/Strings.cs

```diff
@@ -2089,6 +2089,9 @@ public partial struct Settings
         [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
         public static LocalizedString EnableLighting = @"Enable Light Effects";
 
+        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
+        public static LocalizedString EnableMouseScrollZoom = @"Enable Mouse Scroll Zoom";
```

Comment:
It should be `Enable scrolling to change World Scale in game` so that it's clear that it's the same feature